File: boat-bay-frontend/src/app/models/boat-service.ts
import { BoatLintReport } from "./boat-lint-report";
import { BoatStatistics } from "./boat-dashboard";

export interface BoatService {
  id: number;
  key: string;
  name: string;
  content?: string;

  createdOn: Date;
  createdBy: String;

  lastLintReport?: BoatLintReport;

  statistics: BoatStatistics;
}

File: boat-bay-server/src/main/java/com/backbase/oss/boat/bay/web/views/dashboard/models/BoatService.java
package com.backbase.oss.boat.bay.web.views.dashboard.models;

import com.backbase.oss.boat.bay.service.statistics.BoatStatistics;
import java.time.LocalDateTime;
import lombok.Data;

@Data
public class BoatService {


    private String key;
    private String name;
    private String description;
    private String icon;
    private String color;
    private LocalDateTime createdOn;
    private String createdBy;

    private BoatStatistics statistics;

}
